ui/src/common/components/__tests__/CitationSummaryGraph.test.jsx

```diff
@@ -351,4 +368,28 @@ describe('CitationSummaryGraph', () => {
     onCiteableBarUnHover();
     expect(wrapper.state('hoveredBar')).toEqual(null);
   });
+  it('sets graphWidth using ref width and recalculates graphWidth after resize event', () => {
+    const onSelectBarChange = jest.fn();
+    Element.prototype.getBoundingClientRect = jest
```

Comment:
whenever you're doing this in single test case, you need to restore the original `getBoundingClientRect` back at the end